### DISCUSSION TITLE
Customizing select popover sizing and position

### DISCUSSION THREAD

**Discussion by alvarlagerlof** [ariakit/ariakit#1905] [Questions] [answered]
I want to use a Select to make one of those bottom mobile modals with a list of items to select form, but it seems the library includes some styles that are not to my benefit on a div role="presentation". Can I access/restyle this element in any way?

**Comment by diegohaz** [comment, marked as answer]
Yes. You can overwrite the default behavior by passing a memoized `renderCallback` function to `useSelectState`. Here's an example: https://ariakit.org/examples/popover-responsive

**Reply by alvarlagerlof** [reply]
That worked great! Thank you.
